src/elli/dispersions/base_dispersion.py

```diff
@@ -93,9 +92,10 @@ def dielectric_function(self, lbda: npt.ArrayLike) -> npt.NDArray:
 
     def get_mat(self):
         """Returns this dispersion as an isotropic material"""
-        return materials.IsotropicMaterial(self)
+        from ..materials import IsotropicMaterial
```

Comment:
I know this is really ugly, but i think it is the only way to avoid the circular import.